src/bootstrap/src/core/config/config.rs

```diff
@@ -1314,7 +1314,31 @@ impl Config {
 
         // Set flags.
         config.paths = std::mem::take(&mut flags.paths);
-        config.skip = flags.skip.into_iter().chain(flags.exclude).collect();
+        config.skip = flags
+            .skip
+            .into_iter()
+            .chain(flags.exclude)
+            .map(|p| {
+                let p = if cfg!(windows) {
+                    PathBuf::from(p.to_str().unwrap().replace('/', "\\"))
+                } else {
+                    p
+                };
+
+                // Jump to top-level project path to support passing paths
+                // from sub directories.
+                let top_level_path = config.src.join(&p);
+                if !config.src.join(&top_level_path).exists() {
+                    eprintln!("WARNING: '{}' does not exist.", top_level_path.display());
+                }
```

Comment:
Suggestion: this doesn't account for test suite aliases, e.g. `coverage-map` that do not immediately correspond to `tests/<immediate-subdirectory>`. Could you slightly expand the message to say something like

```
WARNING: `{}` is not a valid path or test suite alias
```

to make it more accurate?

Reply:
> WARNING: `{}` is not an existing path or known test suite alias

How do you know it's an unknown alias?

Reply:
... good question. Nevermind